### PR TITLE
hardcode restful api

### PR DESCRIPTION
**Pull request by UlaSubkowska**
working restful api with hardcode data

### CODE REVIEW

**Comment by Marcin-Duszynski** [comment]
pull request is too big. Next time, please split it into a few smaller.

File: backend/src/main/java/blog/entites/Post.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package blog.entites;
+
+public class Post {
```
Comment (Marcin-Duszynski) [inline backend/src/main/java/blog/entites/Post.java]: Why in this class are you not using lombok?

File: backend/src/main/java/blog/services/CommentsServicesProvider.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    @Override
+    public List<Comment> getAllComments() {
+        return comments;
+    }
+
+    @Override
+    public List<Comment> getPostComments(int id) {
+        List<Comment> postComments = new ArrayList<>();
+        for (Comment comment: comments) {
+            if (comment.getPostId()==id){
```
Comment (Marcin-Duszynski) [inline backend/src/main/java/blog/services/CommentsServicesProvider.java]: Please add space before and after ==

File: backend/src/main/java/blog/services/PhotosServicesProvider.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    @Override
+    public List<Photo> getAllPhotos() {
+        return photos;
+    }
+
+    @Override
+    public List<Photo> getAlbumPhotos(int id) {
+        List<Photo> albumPhotos = new ArrayList<>();
+        for (Photo photo: photos) {
+            if (photo.getAlbumId()==id){
```
Comment (Marcin-Duszynski) [inline backend/src/main/java/blog/services/PhotosServicesProvider.java]: Did you check code style using Java Lint tool?
https://marketplace.visualstudio.com/items?itemName=faustinoaq.javac-linter

File: backend/src/main/java/blog/controllers/AlbumsController.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@AllArgsConstructor
+@RestController
+public class AlbumsController {
+
+    private AlbumsServices albumsServices;
```
[the remark this answers is not in the document]
Comment (dzejkobjj) [reply backend/src/main/java/blog/controllers/AlbumsController.java]: The "@AllArgsConstructor" annotation makes constructor with all class filed, in this case those 3 services. It also adds "@Inject" annotation to constructor so Spring knows it should inject those services into this class.

File: backend/src/main/java/blog/entites/Album.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
+package blog.entites;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class Album {
+    private int userId;
```
[the remark this answers is not in the document]
Comment (dzejkobjj) [reply backend/src/main/java/blog/entites/Album.java]: "@Getter" annotation on class creates getters for all class fields, we also added "@Setter" in new commit

**Pull request merged**